graphql/queries.ts

```diff
@@ -76,6 +76,42 @@ export const GET_SUBMISSIONS = gql`
   }
 `
 
+export const ACCEPT_SUBMISSION = gql`
+  mutation acceptSubmission(
+    $submissionId: String!
+    $comment: String!
+    $reviewer: Int!
+  ) {
+    acceptSubmission(
+      id: $submissionId
+      comment: $comment
+      reviewer: $reviewer
+    ) {
+      id
+      comment
+      status
+    }
+  }
+`
+
+export const REJECT_SUBMISSION = gql`
```

Comment:
I think its better to have 1 mutation, `REVIEW_SUBMISSION` with a status `approve / reject`.

Reply:
I felt that it made more sense to have one for each specific status that way we don't have to rely on knowing the particular status string for each one.

Reply:
👍